src/main/resources/public/js/app.js

```diff
@@ -193,41 +193,67 @@ $( '#TheirBoard td'  ).css("background-color", "#42A5F5");
 
 if(gameModel.scanResult == 1){
 
-    document.getElementById("mBox").innerHTML = "ALERT: Scan found at least one ship!"
-    document.getElementById("mBox").style.backgroundColor = "#FDD835";
+    document.getElementById("mBox").innerHTML = "Scan found at least one ship!"
+    document.getElementById("mBox").style.color = "black";
+    document.getElementById("mBox").style.borderColor = "#FDD835";
 
 } else if (gameModel.scanResult == 0){
 
-    document.getElementById("mBox").innerHTML = "ALERT: Scan found no ships!"
-    document.getElementById("mBox").style.backgroundColor = "#FDD835";
-
+    document.getElementById("mBox").innerHTML = "Scan found no ships!"
+    document.getElementById("mBox").style.color = "black";
+    document.getElementById("mBox").style.borderColor = "#e7e7e7";
 }
 
 if(gameModel.validPlace == 1){
 
-    document.getElementById("mBox").innerHTML = "ALERT: Invalid ship placement!";
-    document.getElementById("mBox").style.backgroundColor = "#FDD835";
+    document.getElementById("mBox").innerHTML = "Invalid ship placement!";
+    document.getElementById("mBox").style.color = "#E64A19";
+    document.getElementById("mBox").style.borderColor = "#e7e7e7";
 
     gameModel.validPlace = 0;
 }
 
 if(gameModel.AllShipsPlaced>1)
 {
-    document.getElementById("mBox").innerHTML = "ALERT: All shiprs have been placed already!";
-    document.getElementById("mBox").style.backgroundColor = "#FDD835";
+    document.getElementById("mBox").innerHTML = "All ships have been placed already!";
```

Comment:
I would suggest making a helper function that has these in there. 
document.getElementById("mBox").innerHTML 
document.getElementById("mBox").style.color 
document.getElementById("mBox").style.borderColor 
all of the messages for the text box all share this but with different values for innerHTML, background color and border color.  It seems like a small function to write but I feel like it is one that would be used a lot and could make coding for future messages in text box easier to write.